pkgs/by-name/pe/peergos/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+, jre
+, makeWrapper
+}:
+
+let
+  pname = "peergos";
+  version = "0.14.1";
+  meta = with lib; {
+    description = "A p2p, secure file storage, social network and application protocol";
+    homepage = "https://peergos.org/";
+    # peergos have agpt3 license, peergos-web-ui have gpl3, both are used
+    license = [licenses.agpl3Only licenses.gpl3Only];
+    platforms = platforms.all;
+    maintainers = with maintainers; [ raspher ];
+  };
+  peergos = fetchurl {
+    url = "https://github.com/Peergos/web-ui/releases/download/v${version}/Peergos.jar";
+    hash = "sha256-oCsUuFxTAL0vAabGggGhZHaF40A5TLfkT15HYPiKHlU=";
+  };
+in stdenv.mkDerivation rec {
+  inherit meta peergos pname version;
```

Comment:
This is all pretty minor feeling IMHO, but: 
- You probably don't want to inherit `peergos` as a property of the derivation (no need for that in order to use the sources below).
- You could move `pname` and `meta` back into the body of the derivation just leaving `version` and `peergos` in your `let` block since `peergos` is the only one that doesn't need to be a derivation property and `version` is the only variable the `fetchurl` call depends on.